### ISSUE TITLE
Add `overflow-y: hidden;` to css style `mjx-container`

### ISSUE THREAD

**Issue by thehappycheese** [mitmproxy/pdoc#674]
#### Problem Description

A clear and concise description of the problem you are trying to solve.

Currently `math.html.jinja2` sets `overflow-x: auto;` [see this permalink](https://github.com/mitmproxy/pdoc/blob/7b0745211f701f0ea6e9bc94a62c3d2c0c15a5eb/pdoc/templates/math.html.jinja2#L26C9-L26C26)

In my version of Chrome/Edge has the unintended(?) effect of causing unnecessary scroll bars to appear on the vertical / y axis. To counteract this behaviour we can set `overflow-y: hidden;`

Note that this change should not actually hide anything, since the height of the container is already correctly set by mathjax.

#### Proposal
Change the style in [math.html.jinja2](https://github.com/mitmproxy/pdoc/blob/7b0745211f701f0ea6e9bc94a62c3d2c0c15a5eb/pdoc/templates/math.html.jinja2#L26C9-L26C26) to below

```css
mjx-container {
    overflow-x: auto;
    overflow-y: hidden;
}
```

#### Alternatives

Not sure

#### Additional context

Before the proposed change

![image](https://github.com/mitmproxy/pdoc/assets/2748120/a613a215-e8d7-4d77-af8b-93a69df8d89f)

After the proposed change

![image](https://github.com/mitmproxy/pdoc/assets/2748120/af15e05e-c88f-4b85-9456-dbc95bc38bf9)

**Comment by thehappycheese** [comment]
I will try to raise a PR when I get a chance
